PowerEditor/src/winmain.cpp

```diff
@@ -161,7 +161,7 @@ void parseCommandLine(TCHAR * commandLine, ParamVector & paramVector) {
 	//the commandline generic_string is now a list of zero terminated strings concatenated, and the vector contains all the substrings
 }
 
-bool isInList(const TCHAR *token2Find, ParamVector & params)
+bool removeIfExists(const TCHAR *token2Find, ParamVector & params)

```

Comment:
`isInList` does not do what it says. You expect something immutable, not something that will change the content of the input. In that regard, this function is badly named.